### PR TITLE
[Merged by Bors] - chore: split Algebra.Bounds and Algebra.Order.Pointwise

### PR DESCRIPTION
**Pull request by YaelDillies**
Delete `Algebra.Bounds` and `Algebra.Order.Pointwise`. Create
* `Algebra.Order.Field.Pointwise` for lemmas about pointwise operations in linear ordered fields
* `Algebra.Order.Group.CompleteLattice` for distributivity of group operations over supremum/infimum
* `Algebra.Order.Group.Pointwise.Bounds` for boundedness of pointwise-defined sets
* `Algebra.Order.Group.Pointwise.CompleteLattice` for lemmas about the supremum/infimum of pointwise-defined sets.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Order/Group/CompleteLattice.lean [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Authors: Yury G. Kudryashov
+-/
+import Mathlib.Algebra.Order.Group.OrderIso
+import Mathlib.Order.ConditionallyCompleteLattice.Basic
+
+/-!
+# Upper/lower bounds in ordered monoids and groups
+
+In this file we prove a few facts like “`-s` is bounded above iff `s` is bounded below”
+(`bddAbove_neg`).
```
Comment (Ruben-VandeVelde) [inline Mathlib/Algebra/Order/Group/CompleteLattice.lean:13]: This seems to be about another file

**Review by Ruben-VandeVelde — COMMENTED** [review]
It seems like this PR grew bigger than the PR title/summary suggest. Please update them

**Review by Ruben-VandeVelde — APPROVED** [review]
maintainer merge

**Comment by kim-em** [comment]
bors merge

**Pull request closed without merging**